esp-hal/src/gpio/mod.rs

```diff
@@ -86,11 +86,19 @@ mod placeholder;
 pub use placeholder::NoPin;
 
 #[cfg(soc_etm)]
-pub mod etm;
+crate::unstable_module! {
```

Comment:
Did these really need separate macro calls?

Reply:
the way the macro currently works - yes

I don't see a way to make the macro work here without turning it into a proc-macro (and I'm not sure it's worth it)

Reply:
What do you mean, what's wrong with this?

```rust
crate::unstable_module! {
    #[cfg(soc_etm)]
    pub mod etm;

    #[cfg(lp_io)]
    pub mod lp_io;

    #[cfg(all(rtc_io, not(esp32)))]
    pub mod rtc_io;
}
```

Reply:
mhhh - I remember it was causing some problem with the lp_gpio macro before but now it's working this way - so we can change to one macro invocation again